app/controllers/application_controller.rb

```diff
@@ -4,6 +4,9 @@
 
 class ApplicationController < ActionController::Base
   include Clearance::Controller
+  include Pundit::Authorization
+  after_action :verify_pundit_authorization, except: %i[health_check new create]
```

Comment:
After thinking about this for a while, it would probably be best to not exclude `new` and `create` here. Rubocop doesn't like it anyway.

Removing it from here, you will need to skip authorization for in the recipe and user controllers.

Reply:
So would do I just disable the line then? Leave them there.

Reply:
Never mind, I missunderstood what you meant.